function/examples/README.md

```diff
@@ -1,8 +1,37 @@
-# Examples 
+# function-examples
 
 ## Overview
 
-The following examples show how to use Function API to:
+This module contains examples showing how to use Function API.
 
-* [Initialize a workspace](./init/README.md)
-* [Apply a Function to a cluster](./apply/README.md)
+## Operators
+
+Operators allow to perform two operations on given resources: `apply` and `delete`. 
+Both operations may accept callbacks that will be executed before and/or after each operation.
+
+The library contains two implementations of operator interface: `GenericOperator` and `TriggersOperator`.
+
+Examples:
+
+* [apply operation](./cmd/operator/apply/main.go)
```

Comment:
```suggestion
* [Apply operation](./cmd/operator/apply/main.go)
```

Reply:
Simply because we [capitalize first letters in bullet points](https://kyma-project.io/community/guidelines/content/#formatting-formatting-ordered-and-unordered-lists).